charts/tempo-distributed/templates/admin-api/admin-api-dep.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.enterprise.enabled -}}
+{{ $dict := dict "ctx" . "component" "admin-api" "memberlist" true }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    {{- toYaml .Values.adminApi.annotations | nindent 4 }}
+  labels:
+    {{- include "tempo.labels" (dict "ctx" . "component" "admin-api" "memberlist" true) | nindent 4 }}
```

Comment:
why not use the same `$dict` here (as you're using below)?

Reply:
Good catch.